### PR TITLE
Login-Signup---designed

### PR DESCRIPTION
**Pull request by itsmebipul00**
Login, signup page --designed
Timer, endgame --added

### CODE REVIEW

**Review by Shubhamkr2610 — APPROVED** [review]
every thing looks good to me. you can merge it

**Review by DivyanshSareen — APPROVED** [review]
good to merge

**Review by Muzammil-khan710 — APPROVED** [review]
Good to merge bipul !

File: form.css [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+.form-login {
+  width: 40%;
+  margin: 0 auto;
+}
+.already-done {
+  margin-top: 0.5rem;
```
Comment (zoyatanweer) [inline form.css:13]: write in px

**Review by zoyatanweer — COMMENTED** [review]
code is clean. UI suggestion :
On hovering on the "wildlife category", the text "wildlife" is black, and since it is black it gets blended with the background so it is not standing out. See if you can try other colors for all the category.
Rest all looks good, good to merge.

**Pull request merged**